integrationtests/tests/fixtures/mod.rs

```diff
@@ -848,6 +858,9 @@ impl FederationTest {
                 dbtx.commit_tx().await.expect("DB Error");
             });
         }
+        bitcoin
+            .mine_blocks(user.client.wallet_client().config.finality_delay as u64)
+            .await;
```

Comment:
Not sure how things worked without this before...but it's necessary now for the tests to pass.